hack/deploy-cert-manager.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+source hack/common.sh
+
+set -ex
+
+CERT_MANAGER_TIMEOUT=${CERT_MANAGER_TIMEOUT:-"120s"}
+
+echo "Installing cert-manager ${CERT_MANAGER_VERSION}"
+${CMD} apply -f https://github.com/cert-manager/cert-manager/releases/download/${CERT_MANAGER_VERSION}/cert-manager.yaml
+
+echo "Waiting for cert-manager to be ready..."
+${CMD} wait --for=condition=Available --namespace=cert-manager deployment/cert-manager --timeout=${CERT_MANAGER_TIMEOUT}
+${CMD} wait --for=condition=Available --namespace=cert-manager deployment/cert-manager-cainjector --timeout=${CERT_MANAGER_TIMEOUT}
+${CMD} wait --for=condition=Available --namespace=cert-manager deployment/cert-manager-webhook --timeout=${CERT_MANAGER_TIMEOUT}
+
+echo "Waiting for cert-manager webhook to be ready..."
+${CMD} wait --for=condition=Ready --namespace=cert-manager pod -l app=webhook --timeout=${CERT_MANAGER_TIMEOUT}
+
+echo "Waiting for cert-manager CRDs to be available..."
+${CMD} wait --for condition=established --timeout=${CERT_MANAGER_TIMEOUT} crd/certificates.cert-manager.io
+${CMD} wait --for condition=established --timeout=${CERT_MANAGER_TIMEOUT} crd/issuers.cert-manager.io
```

Comment:
Don't we need the CRDs to be available before we check for the pods/deployments?